jwst/lib/set_telescope_pointing.py

```diff
@@ -194,6 +195,11 @@ def add_wcs(filename, default_pa_v3=0., siaf_path=None, engdb_url=None,
         reduce_func=reduce_func,
         **transform_kwargs
     )
+
+    if model.meta.target.type:
```

Comment:
For safety's sake, should this have a check for None or AttributeError around it, just in case the input doesn't have the TARGTYPE keyword? That should be a pretty basic keyword (and hence in every "real" dataset coming through DMS), but you can never be too careful (IMHO).